content/integrations/new_relic.md

```diff
@@ -6,36 +6,39 @@ git_integration_title: new_relic
 newhlevel: true
 ---
 
-## Overview
-
 {{< img src="integrations/new_relic/newrelicdashboard.png" alt="New Relic Dashboard" >}}
 
+## Overview
+
 Connect to New Relic to:
 
 * See key New Relic metrics (like response time and Apdex score) in context with the rest of your Datadog metrics
```

Comment:
How about:

* See key New Relic metrics (like response time and Apdex score) in context with the rest of your Datadog metrics (**only works with New Relic Pro accounts and above**)

Then remove the **Notice** below?